### PR TITLE
Update the French Translation

### PR DESCRIPTION
**Pull request by itskatt**

### CODE REVIEW

File: core/assets/bundles/bundle_fr.properties [hunk 1]
```diff
@@ -99,6 +99,7 @@ mod.enabled = [lightgray]Activé
 mod.disabled = [scarlet]Désactivé
 mod.disable = Désactiver
 mod.delete.error = Unable to delete mod. File may be in use.
+mod.requiresversion = [scarlet]Version du jeu requise: [accent]{0}
```
Comment (YaFou) [inline core/assets/bundles/bundle_fr.properties]: A space must be before a `:`.
```suggestion
mod.requiresversion = [scarlet]Version du jeu requise : [accent]{0}
```

File: core/assets/bundles/bundle_fr.properties [hunk 2]
```diff
@@ -1084,7 +1088,7 @@ mech.alpha-mech.description = Le mécha standard. Est basé sur une unité Poign
 mech.delta-mech.description = Un mécha rapide, avec une armure légère, conçu pour les attaques de frappe. Il inflige, par contre, peu de dégâts aux structures. Néanmoins il peut tuer de grand groupes d'ennemis très rapidement avec ses arcs électriques.
 mech.tau-mech.description = Un mécha de support. Soigne les blocs alliés en tirant dessus. Il peut aussi éteindre les feux et soigner ses alliés en zone avec sa compétence.
 mech.omega-mech.description = Un mécha cuirassé et large fait pour les assauts frontaux. Sa compétence lui permet de bloquer 90% des dégâts.
-mech.dart-ship.description = Le vaisseau standard. Raisonnablement rapide et léger. Il a néanmoins peu d'attaque et une faible vitesse de minage.
+mech.dart-ship.description = Le vaisseau standard. Raisonnablement rapide et léger, et possède une vitesse de minage rapide. Néanmoins, ses capacitées d'attaque sont faibles.
```
Comment (YaFou) [inline core/assets/bundles/bundle_fr.properties]: `et` twice is not allowed. `possède` must have a subject.
```suggestion
mech.dart-ship.description = Le vaisseau standard. Il est raisonnablement rapide, léger et possède une vitesse de minage rapide. Néanmoins, ses capacitées d'attaque sont faibles.
```

**Comment by itskatt** [comment]
@YaFou Thanks for the review!

File: core/assets/bundles/bundle_fr.properties [hunk 3]
```diff
@@ -1084,7 +1088,7 @@ mech.alpha-mech.description = Le mécha standard. Est basé sur une unité Poign
 mech.delta-mech.description = Un mécha rapide, avec une armure légère, conçu pour les attaques de frappe. Il inflige, par contre, peu de dégâts aux structures. Néanmoins il peut tuer de grand groupes d'ennemis très rapidement avec ses arcs électriques.
 mech.tau-mech.description = Un mécha de support. Soigne les blocs alliés en tirant dessus. Il peut aussi éteindre les feux et soigner ses alliés en zone avec sa compétence.
 mech.omega-mech.description = Un mécha cuirassé et large fait pour les assauts frontaux. Sa compétence lui permet de bloquer 90% des dégâts.
-mech.dart-ship.description = Le vaisseau standard. Raisonnablement rapide et léger. Il a néanmoins peu d'attaque et une faible vitesse de minage.
+mech.dart-ship.description = Le vaisseau standard. Il est raisonnablement rapide, léger et possède une vitesse de minage rapide. Néanmoins, ses capacitées d'attaque sont faibles.
```
Comment (Paulao17) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
mech.dart-ship.description = Le vaisseau standard. Il est raisonnablement rapide, léger et possède une vitesse de minage rapide. Néanmoins, ses capacités d'attaque sont faibles.
```

**Pull request merged**